swagger.yaml

```diff
@@ -1513,7 +1515,7 @@ components:
           type: string
           format: date-time
           readOnly: true
-    StoryJson:
+    ExporttedStory:
       type: object
       properties:
         id:
```

Comment:
remove id property

Reply:
Remove readOnly: true properties

Reply:
Move Knots to ExportedStoryKnot, Intents to ExportedStoryIntent, etc..

Reply:
Remove variable storyId, remove variable id

Reply:
Remove script id

Reply:
Move training materials to their own export object

Reply:
Resolved